examples/12. Advanced Action.md

```diff
@@ -0,0 +1,66 @@
+FORMAT: 1A
+
+# Advanced Action API
+An action in a resource in the API Blueprint can be considered as a state transition. This API example demonstrates that a resource can contain a state transition to another resource. This allows us to show that the resource `Task` can afford an action that is not limited to the resource `Task`.
+
+## API Blueprint
++ [Previous: Resource Model](11.%20Resource%20Model.md)
++ [This: Raw API Blueprint](https://raw.github.com/apiaryio/api-blueprint/master/examples/11.%20Advanced%20Action.md)
+
+# Task [/tasks/{id}]
+
++ Parameters
+    + id (string)
+
+## Retrieve Task [GET]
+
++ Response 200 (application/json)
+
+        {
+            "id": 123,
+            "name": "Go to gym",
+            "done": false,
+            "type": "task"
+        }
+
+## Update Task [PATCH]
+This is a state transition within the same resource
+
++ Request (application/json)
+
+        {
+            "name": "Exercise in gym"
+        }
+
++ Response 200 (application/json)
+
+        {
+            "id": 123,
+            "name": "Exercise in gym",
+            "done": false,
+            "type": "task"
+        }
+
+## List All Tasks [GET /tasks{?status,priority}]
```

Comment:
While this is a valid example, I consider it  not the best show case. Usually an item of collection should not be "affording" you with viewing every other item – in my opinion it should be the other way around. A collection should afford you with the action to view, create, delete a single item.